unix/c_editline.c

```diff
@@ -419,6 +471,13 @@ void *go_libedit_gets(EditLine *el, char *lprompt, char *rprompt,
     el_set(el, EL_SETTY, "-q", "susp=", NULL);
     el_reset(el);
 
+    // Request bracketed pastes if supported by the terminal.
+    int bracketed_paste = 0;
+    if (go_libedit_term_supports_bracketed_paste(el)) {
```

Comment:
Compute/store the compat bool so that it doesn't get re-evaluated on every call to gets.

Reply:
Done. (I also added a commit to fix a missing call to set the clientdata.)